src/main/java/org/opentestsystem/ap/ims/service/ItemManagementService.java

```diff
@@ -101,8 +101,8 @@ public void syncItemToItemBank(String itemId) {
      */
     @Transactional(readOnly = true)
     public List<WorkflowStatus> findAllWorkflowStatus() {
-        itemManager.findAllWorkflowStatuses();
-        return itemManager.findAllWorkflowStatuses();
+        List<WorkflowStatus> statuses = itemManager.findAllWorkflowStatuses();
+        return statuses;
```

Comment:
For some reason this method was fetching workflow statuses twice.  I left the two line call assuming someone did this at one point for debugging.